charts/csm-installer/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: csm-installer
-version: 1.0.2
-appVersion: 1.0.2
+version: 1.0.1
```

Comment:
question: Why is there a downgrade in version?

Reply:
This chart was incremented in the 1.10.2 release but this chart is deprecated so a new version should not have been released. I'm not sure if this will impact the current release by downgrading so I will revert it back to 1.0.2.

Reply:
Fixed in #473